docs/dev/migrating-to-v1.rst

```diff
@@ -240,6 +240,7 @@ In the core of HydroMT, the available components are:
 | model.tables          | TablesComponent  | Component for managing non-geospatial data in pandas DataFrames  |
 | model.grid            | GridComponent   | Component for managing regular gridded data in single hydromt RasterDataset    |
 | model.geoms('region') | RegionComponent | Component for managing the area of interest for the model in a geopandas GeoDataFrame.    |
+| model.mesh			| MeshComponent	  | Component for managing unstructured grids as a hydromt RasterDataset
```

Comment:
Table format looks off. Can you check? All columns length should match ie "|" should be aligned. If it was because of my suggestion then it's because I cant easily check alignment in this github view sorry..